core/domain/suggestion_services.py

```diff
@@ -165,6 +165,22 @@ def _update_suggestion(suggestion):
     suggestion_model.put()
 
 
+def edit_suggestion(suggestion_id, new_change):
+    """Edits the submitted suggestion.
+
+    Args:
+        suggestion_id: obj. Id of suggestion to be changed.
+        new_change: dict(str, str). Dictionary contains the change in
+        suggestion.
```

Comment:
You'll need to add an indent to this line I believe? (/cc @oppia/dev-workflow-team)

Reply:
done

Reply:
An issue already exists #6422 for this.
Thanks!